tests/unit/source_code/test_notebook_linter.py

```diff
@@ -154,6 +154,22 @@
 MERGE INTO delta.`/dbfs/...` t USING source ON t.key = source.key WHEN MATCHED THEN DELETE
     """,
             [
+                Deprecation(
+                    code='cloud-access',
+                    message='The use of cloud direct references is deprecated: ' "'dbfs:/mnt/foo/bar'",
```

Comment:
```suggestion
                    message='The use of cloud direct references is deprecated: 'dbfs:/mnt/foo/bar'",
```
there's a bug somewhere. ` "'` got into the string but it shouldn't be there.

Reply:
Hmm - I will track that down

Reply:
Quotes in advice messages no longer present